jumanji/wrappers_test.py

```diff
@@ -58,6 +59,12 @@ class MockWrapper(Wrapper[FakeState]):
     return MockWrapper
 
 
+@pytest.fixture
+def fake_state() -> FakeState:
+    return FakeState(key=jax.random.PRNGKey(0),
+                     step=jax.random.randint(jax.random.PRNGKey(0), [1], 0, 100))
+
+
```

Comment:
If the fake state is used in only one test, it may be preferable not to make it a fixture and define it directly in the corresponding test. Also, I believe the step argument must rather be `step=jax.random.randint(jax.random.PRNGKey(0), (), 0, 100)`.

Reply:
I would also change the key so that it's not the same as the other key defined in the `test_jumanji_environment_to_deep_mind_env__state_getter_setter` test.